### ISSUE TITLE
Mockup help widget

### ISSUE THREAD

**Issue by philipmjohnson** [radgrad/radgrad#46]
* Please do all work for this issue in branch issue-46.
* Design a collapsible, dismissible help pane below menu area to provide context-sensitive, per-page help on the UI.

**Comment by philipmjohnson** [comment]
Branch issue-46 has an initial mockup for review. I started with the Student view. Note that the left hand menu is now shorter: there's no need for an "About ICE" or "About Levels" page, because this information is now integrated into the help pane.

Here's the student home page with the help pane collapsed. That's the default:

![student-home](https://cloud.githubusercontent.com/assets/290288/20870161/37885a6c-ba27-11e6-8bfd-896c98088bde.png)

Here's the home page with the help pane expanded:

![student-home-expanded](https://cloud.githubusercontent.com/assets/290288/20870163/4beeca7c-ba27-11e6-8e91-c1949eecce47.png)

Here's the ICE Page by default:

![student-ice](https://cloud.githubusercontent.com/assets/290288/20870168/5ab9a4e6-ba27-11e6-9177-30bb04039a91.png)

And now here is the ICE page with the help. I just inserted the "About ICE" page into the help pane. We might want to revise this, but you get the idea:

![student-ice-expanded](https://cloud.githubusercontent.com/assets/290288/20870171/70aa9814-ba27-11e6-8c93-d81da333d30e.png)
